src/processing.rs

```diff
@@ -79,44 +79,49 @@ pub fn stack_frames(
     );
 
     // Initial necessary values
-    length_signal = sig.shape[0];
-    frame_sample_length = int(np.round(sampling_frequency * frame_length)); // Defined by the number of samples
-    frame_stride = (sampling_frequency * frame_stride).round();
+    let length_signal = sig.shape[0];
+    let frame_sample_length = (sampling_frequency * frame_length).round(); // Defined by the number of samples
+    let frame_stride = (sampling_frequency * frame_stride).round();
+    let mut len_sig = 0;
+    let mut numframes = 0;
 
     // Zero padding is done for allocating space for the last frame.
-    if zero_padding {
+    let signal = if zero_padding {
         // Calculation of number of frames
-        numframes = (int(math.ceil((length_signal - frame_sample_length) / frame_stride)));
-        println!(numframes, length_signal, frame_sample_length, frame_stride);
+        numframes = ((length_signal - frame_sample_length) / frame_stride).ceil() as i32;
+        println!(
+            "{} {} {} {}",
+            numframes, length_signal, frame_sample_length, frame_stride
+        );
 
         // Zero padding
-        len_sig = int(numframes * frame_stride + frame_sample_length);
-        additive_zeros = np.zeros((len_sig - length_signal,));
-        signal = np.concatenate((sig, additive_zeros));
+        len_sig = (numframes * frame_stride + frame_sample_length) as i32;
+        let additive_zeros = ndarray::ArrayBase::zeros((len_sig - length_signal,));
```

Comment:
This would look better if we Array1 or Array2 was used, more explicit and easier to read than ArrayBase.

Reply:
yeah, on the Array1 or 2 thing, I'm mainly defaulting to arrayBase because I'm unsure of the expected dimensions and if the dimensions change for some operations. 

would it be better to use 1 or 2 from the get go or replace 1 or 2 in another iteration, once we can log the structs data at runtime?

Reply:
what dimensions is additive_zeros supposed to be?